tests/test_generate_mbz_toc.py

```diff
@@ -283,3 +285,66 @@ def test_toc_creation_page_and_lesson_together(
         assert lines[3] == \
             f'        * [{lesson_page_name}](./html/{html_filenames["L"]})'
         assert lines[4] == f'    * [{page_name}](./html/{html_filenames["P"]})'
+
+
+def test_toc_creation_page_and_lesson_together_csv(
+    mocker, tmp_path, mbz_builder, lesson_builder, page_builder
+):
+    lesson_html = "<p>Lesson Content</p>"
+    lesson_name = "Only Lesson"
+    lesson_page_name = "Lesson Page 1"
+    lesson = lesson_builder(
+            id=1,
+            name=lesson_name,
+            pages=[
+                {
+                    "id": "1",
+                    "title": lesson_page_name,
+                    "html_content": lesson_html
+                }
+            ]
+    )
+    page_html = "<p>Page Content</p>"
+    page_name = "Only Page"
+    page = page_builder(
+            id=1,
+            name=page_name,
+            html_content=page_html
+        )
+
+    mbz_path = tmp_path / "mbz"
+    html_path = tmp_path / "html"
+    html_path.mkdir()
+    mbz_builder(mbz_path, activities=[lesson, page])
+    mbtools.extract_html_content.replace_content_tags(mbz_path, html_path)
+    md_filepath = tmp_path
+
+    mocker.patch(
+        "sys.argv",
+        ["", f"{mbz_path}", str(md_filepath), '--csv']
+    )
+    main()
+
+    html_filenames = {}
+    for path, dirs, files in os.walk(html_path):
+        assert len(files) == 2
+        for html_filename in files:
+            with open((html_path / html_filename), 'r') as f:
+                content = f.read()
+                if content == lesson_html:
+                    html_filenames["L"] = html_filename
+                elif content == page_html:
+                    html_filenames["P"] = html_filename
```

Comment:
This seems like irrelevant code that can be deleted. Perhaps from a copy / paste?